src/content/main.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <script>var id = {{id}}</script>
+    <script>{{optional_create_window_callback}}</script>
```

Comment:
Would it be better to put the `<script></script>` tags in the mustache template as well so that there isn't an empty `script` node sitting around if the callback isn't being used?
I had it this way originally, because i think it makes the .html file more readable: you know that's going to be a `script` node, no matter what its contents end up being.